extras/jwt/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../kustomize-base
```

Comment:
Question:
As part of the larger _Kustomize + Bank of Anthos_ project, we will preserve the `/kubernetes-manifests/` folder until all tutorials/content have been updated to use `kubectl -k` and this `/kustomize-base/` folder.
So this `/kustomize-base/` folder or the `/extras/jwt` folder will be the go-to folder for deployments of Bank of Anthos.

Reply:
While the kustomize migration is in progress, would it be possible to symlink the kubernetes-manifests YAML files from the base directory?

Reply:
Personally I prefer using the Bases and Overlays structure of  `kustomize/base` and `kustomize/overlays` to keep all the YAML and Kustomize in one tree.

Reply:
@arueth Thanks for the tips!

More info about the Kustomize integration project can be found in [this Google-internal document](https://docs.google.com/document/d/1asQ13-gSVXiYhmIA6qXvll26RdBU0j-Iv1b4dybYuPQ/edit?resourcekey=0-tofFFgfmwMh1tZnv_zr7Gg#heading=h.q2kc8tkcg5a2).

**1. Symbolic Link**
Oo, that's a very clever idea!
I've never actually used symbolic links in a git repository before.
I've added your idea to our planning doc. :)
I'll run it by Olivier and Mathieu today.

**2. `kustomize/overlays` Folder Structure**
My plan is to have the overlays that belong to each "extra" in the [associated "extra" sub-folder](https://github.com/GoogleCloudPlatform/bank-of-anthos/tree/release/v0.5.4/extras).
I'm hesitant to disrupt the folder structure of Bank of Anthos — given how much content uses Bank of Anthos.
But I will definitely ask @bourgeoisor today about the folder structure you're suggesting.

Reply:
**Update:** We're now using the `kustomize/overlays` folder structure in this pull-request.